pkg/reconciler/v1alpha1/revision/table_test.go

```diff
@@ -1683,13 +1694,19 @@ func addKPAStatus(kpa *kpav1alpha1.PodAutoscaler, status kpav1alpha1.PodAutoscal
 
 // Build is a special case of resource creation because it isn't owned by
 // the Revision, just tracked.
-func build(namespace, name string, conds ...duckv1alpha1.Condition) *buildv1alpha1.Build {
-	return &buildv1alpha1.Build{
-		ObjectMeta: om(namespace, name),
-		Status: buildv1alpha1.BuildStatus{
-			Conditions: conds,
-		},
-	}
+func build(namespace, name string, conds ...duckv1alpha1.Condition) *unstructured.Unstructured {
+	b := &unstructured.Unstructured{}
+	b.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "build.knative.dev",
+		Version: "v1alpha1",
+		Kind:    "Build",
+	})
+	b.SetName(name)
+	b.SetNamespace(namespace)
+	b.Object["status"] = map[string]interface{}{"conditions": conds}
+	u := &unstructured.Unstructured{}
+	duck.FromUnstructured(b, u)
```

Comment:
I don't understand the need for this?

Reply:
To prevent a panic in `(*Unstructured).DeepCopy()`

Adding a code comment.